sample/Sample.iOS/AppDelegate.cs

```diff
@@ -30,6 +30,7 @@ public override bool FinishedLaunching(UIApplication uiApplication, NSDictionary
             NotifoIO.Current
                 .SetNotificationHandler(new NotificationHandler())
                 .UseDefaults()
+                .SetRefreshOptions(new PullRefreshOptions())
```

Comment:
Is this needed?

Reply:
Would be better to set it to a default instance in the implementation.

Reply:
it is, else you get a null reference in PullPendingNotificationsAndTrack()

Reply:
but I agree makes more sense to add it in the implementation